lib/ex_doc/formatter/html/templates/head_template.eex

```diff
@@ -16,8 +16,12 @@
     <% end %>
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta name="generator" content="ExDoc v<%= ExDoc.version %>">
-    <link href="http://fonts.googleapis.com/css?family=Lato:400,300,700,900|Merriweather:300italic,300,700,700italic|Inconsolata:400,700" rel="stylesheet" type="text/css">
-    <link rel="stylesheet" href="dist/app.css" />
+    <script>
+      var cdn_protocol;
```

Comment:
@milmazz  @Dignifiedquire is it alright creating this global var, if not just let me know what is the best way to do it.